src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends AbstractFigure {
```

Comment:
Don't use abstract classes to set behavior for classes. Consider whether `Circle` should extend an abstract class or perhaps implement an interface that defines the behavior for figures.